ql/quotes/futuresconvadjustmentquote.hpp

```diff
@@ -65,11 +66,13 @@ namespace QuantLib {
         Handle<Quote> futuresQuote_;
         Handle<Quote> volatility_;
         Handle<Quote> meanReversion_;
+        mutable Real rate_ = Null<Real>();
```

Comment:
Too late for the others, but this one can be private :)

Reply:
Oops, I'm inheriting from it to implement a different convexity model :)

Reply:
Ok then 😆

Reply:
Thank you!